app/src/main/java/chat/rocket/android/chatroom/viewmodel/MessageViewModel.kt

```diff
@@ -56,36 +60,55 @@ data class MessageViewModel(val context: Context,
         }
     }
 
-    fun getAvatarUrl(serverUrl: String): String? {
-        return message.sender?.username.let {
-            return@let UrlHelper.getAvatarUrl(serverUrl, it.toString())
+    private fun getUserAvatar(): String? {
+        val username = message.sender?.username ?: "?"
+        return baseUrl?.let {
+            UrlHelper.getAvatarUrl(baseUrl.value.toString(), username)
         }
     }
 
-    fun getTime() = DateTimeHelper.getTime(DateTimeHelper.getLocalDateTime(message.timestamp))
+    private fun getTime() =
+        DateTimeHelper.getTime(DateTimeHelper.getLocalDateTime(message.timestamp))
 
-    fun getSenderName(): CharSequence {
+    private fun getSender(): CharSequence {
         val useRealName = settings?.get(USE_REALNAME)?.value as Boolean
         val username = message.sender?.username
         val realName = message.sender?.name
         val senderName = if (useRealName) realName else username
-        return senderName ?: username.toString()
+        return senderName ?: context.getString(R.string.msg_unknown)
     }
 
-    fun getContent(context: Context): CharSequence {
+    private fun getContent(context: Context): CharSequence {
         val contentMessage: CharSequence
         when (message.type) {
-        //TODO: Add implementation for Welcome type.
-            is MessageRemoved -> contentMessage = getSystemMessage(context.getString(R.string.message_removed))
-            is UserJoined -> contentMessage = getSystemMessage(context.getString(R.string.message_user_joined_channel))
-            is UserLeft -> contentMessage = getSystemMessage(context.getString(R.string.message_user_left))
-            is UserAdded -> contentMessage = getSystemMessage(
-                    context.getString(R.string.message_user_added_by, message.message, message.sender?.username))
-            is RoomNameChanged -> contentMessage = getSystemMessage(
-                    context.getString(R.string.message_room_name_changed, message.message, message.sender?.username))
-            is UserRemoved -> contentMessage = getSystemMessage(
-                    context.getString(R.string.message_user_removed_by, message.message, message.sender?.username))
-            else -> contentMessage = getNormalMessage()
+            //TODO: Add implementation for Welcome type.
+            is MessageRemoved -> {
```

Comment:
For one line statements it is ok to go inline and leave the `{}` out (https://kotlinlang.org/docs/reference/coding-conventions.html#method-call-formatting)

Reply:
Just make this change because it is more readable. Do you agree?

Reply:
I actually prefer the one line version